cmake/SearchForStuff.cmake

```diff
@@ -635,7 +659,7 @@ endif ()
 ########################################
 # Find SDFormat
 set(SDF_MIN_REQUIRED_VERSION 9.8)
-find_package(sdformat9 ${SDF_MIN_REQUIRED_VERSION} REQUIRED)
+find_package(sdformat9 CONFIG ${SDF_MIN_REQUIRED_VERSION} REQUIRED)
```

Comment:
Why it is necessary to add `CONFIG`  here?

Reply:
sdformat9 also come from vcpkg. This is how vcpkg search. It search with config. 
I can try without config, but I am not sure how if it work.
The config also work for conda. 
Are you insist on it?

Reply:
I will try to remove it.

Reply:
done.